components/ui/badge.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+interface BadgeProps {
+  children: React.ReactNode;
+  counter: number;
+}
+
+const Badge: React.FC<BadgeProps> = ({ children, counter }) => {
+  const badgeCounter = counter > 99 ? "99+" : counter;
```

Comment:
I don't think it's good if we tailored the Badge for only counter, it should be more generic use case for showing Badge content

Reply:
@ntheanh201 what additional content you're suggesting for example ?

Reply:
@swve for me, change the counter to `content`, the condition should move out of badge component, and pass it through props like this:
`<Badge content={issues.length > 99 ? "99+" : issues.length}>`

Reply:
I agree, we can pass tailwind colours too.

`<Badge tw-color="bg-red-600" content={issues.length > 99 ? "99+" : issues.length}>`

Reply:
Thanks for the suggestions!

@swve 
I made a small change to the idea of `tw-color`. I added a `className` prop, which allows you to add both additional badge colors and new styles.

The default red remains unchanged.

Reply:
LGTM

Reply:
LGTM2